runtime/middleware/jwt.go

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2024 Hypermode Inc.
+ * Licensed under the terms of the Apache License, Version 2.0
+ * See the LICENSE file that accompanied this code for further details.
+ *
+ * SPDX-FileCopyrightText: 2024 Hypermode Inc. <hello@hypermode.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package middleware
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/hypermodeinc/modus/runtime/config"
+	"github.com/hypermodeinc/modus/runtime/logger"
+	"github.com/hypermodeinc/modus/runtime/utils"
+
+	"github.com/golang-jwt/jwt/v5"
+)
+
+type jwtClaimsKey string
+
+const jwtClaims jwtClaimsKey = "jwt_claims"
+
+var PrivKeys map[string]string
```

Comment:
- They are public keys, not private keys
- The field should be lower-cased
- Make it a map of string to parsed key.  Then you can just use the parsed key when needed later instead of parsing with each request.

```suggestion
var jwtKeys = map[string]*rsa.PublicKey
```